interfaces/acados_template/examples/matlab/pendulum_example/export_ode_model.m

```diff
@@ -0,0 +1,54 @@
+function model = export_ode_model()
```

Comment:
Can the code in `interfaces/acados_template/examples/matlab/` removed, or can we put it into `experimental`?

Reply:
good point (addressed here 560bcb7)